### PR TITLE
Having a single CodeCov report for integration tests

### PR DESCRIPTION
**Pull request by davidrabinowitz**

### CODE REVIEW

File: cloudbuild/presubmit.sh [hunk 1]
```diff
@@ -54,14 +59,13 @@ case $STEP in
     $MVN failsafe:integration-test failsafe:verify jacoco:report jacoco:report-aggregate -Pcoverage,integration,dsv2_3.1
     ;;
 
+  upload-it-to-codecov)
```
[the remark this answers is not in the document]
Comment (davidrabinowitz) [reply cloudbuild/presubmit.sh:62]: @sonatype-lift ignore

File: cloudbuild/presubmit.sh [hunk 2]
```diff
@@ -54,14 +59,13 @@ case $STEP in
...
+    checkenv
+    # Upload test coverage report to Codecov
+    bash <(curl -s https://codecov.io/bash) -K -F integrationtest
+
   *)
```
[the remark this answers is not in the document]
Comment (davidrabinowitz) [reply cloudbuild/presubmit.sh:68]: @sonatype-lift ignoreall

**Review by suryasoma — APPROVED** [review]

**Pull request merged**